ade/docker-base/Dockerfile

```diff
@@ -86,6 +86,7 @@ RUN apt-get update \
   && mkdir -p /root/ros2_$ROS_VERSION/src \
   && wget https://raw.githubusercontent.com/ros2/ros2/$ROS_VERSION/ros2.repos \
   && vcs import src < ros2.repos \
+  && git clone https://gitlab.com/ApexAI/apex_test_tools src/apex_test_tools \
```

Comment:
@LanderU I would clone the foxy branch so we can update this easily for each ROS release

Reply:
@carlossvg, done!